source/common/quic/envoy_quic_server_stream.h

```diff
@@ -108,6 +110,8 @@ class EnvoyQuicServerStream : public quic::QuicSpdyServerStreamBase,
   Http::RequestDecoder* request_decoder_{nullptr};
   envoy::config::core::v3::HttpProtocolOptions::HeadersWithUnderscoresAction
       headers_with_underscores_action_;
+  quic::QuicSpdySession* session_;
```

Comment:
Similarly this can be queried from the parent class

Reply:
Done.